service/worker/deployment/deployment_client.go

```diff
@@ -240,6 +250,47 @@ func (d *DeploymentClientImpl) DescribeDeployment(ctx context.Context, namespace
 	}, nil
 }
 
+func (d *DeploymentClientImpl) GetDeploymentReachability(
+	ctx context.Context,
+	namespaceEntry *namespace.Namespace,
+	seriesName string,
+	buildID string,
```

Comment:
Nit: wondering if the this client's methods should get a single `deployment` instead of both these values.

Reply:
yeah, I think that would be nice, I just copied what Shivam did since I wanted them to be the same. Maybe I can send a follow-up PR to change all of them at the same time later?

Reply:
added a todo